src/katsdpmodels/fetch/aiohttp.py

```diff
@@ -94,18 +96,18 @@ class Fetcher(fetch.FetcherBase):
     """
 
     def __init__(self, *,
-                 session: Optional[aiohttp.ClientSession] = None,
+                 session: Optional[ClientSession] = None,
```

Comment:
Given that retries are not optional in the sync code, and the default is the retry client, why allow the normal client at all ?

Reply:
Retries are optional in the sync client - if you supply your own requests.Session then katsdpmodels won't try to impose any retry logic on it.

Allowing the normal client allows for use cases we haven't thought of e.g. maybe the user has extended or configured an aiohttp.ClientSession in some way, or wants to supply their own retry logic.

Reply:
I think that is a bit tenuous - you can just as easily extend the RetryClient if you want something funky, but no point bikeshedding it to death....